#! /bin/sh -e
## home_as_game_dir.dpatch by Andres Mejia <mcitadel@gmail.com>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Patch to set game directory used by client and server from $HOME/.codered to $HOME/.alienarena2007

dir=
if [ $# -eq 3 -a "$2" = '-d' ]; then
    pdir="-d $3"
    dir="$3/"
elif [ $# -ne 1 ]; then
    echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
    exit 1
fi
case "$1" in
    -patch)
        patch $pdir -f --no-backup-if-mismatch -p0 < $0
        ;;
    -unpatch)
        patch $pdir -f --no-backup-if-mismatch -R -p0 < $0
        ;;
    *)
        echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
        exit 1
esac
exit 0

--- source/qcommon/files.c~	2007-05-13 15:31:30.000000000 -0400
+++ source/qcommon/files.c	2007-05-17 16:00:07.000000000 -0400
@@ -633,7 +633,7 @@
 ================
 FS_AddHomeAsGameDirectory
  
-Use ~/.codered/dir as fs_gamedir
+Use ~/.alienarena2007/dir as fs_gamedir
 ================
 */
 void FS_AddHomeAsGameDirectory (char *dir)
@@ -642,7 +642,7 @@
 	char *homedir=getenv("HOME");
 	if(homedir)
 	{
-		int len = snprintf(gdir,sizeof(gdir),"%s/.codered/%s/", homedir, dir);
+		int len = snprintf(gdir,sizeof(gdir),"%s/.alienarena2007/%s/", homedir, dir);
 		Com_Printf("using %s for writing\n",gdir);
 		FS_CreatePath (gdir);
 
